app/models/parsers/bioanalysis_csv_parser.rb

```diff
@@ -1,6 +1,7 @@
 # frozen_string_literal: true
 
-class Parsers::BioanalysisCsvParser # rubocop:todo Style/Documentation
+# rubocop:todo Metrics/ClassLength
+class Parsers::BioanalysisCsvParser # rubocop:todo Style/Documentation, Metrics/ClassLength
```

Comment:
There's some weird duplication of the `:todo` markers here.  I assume those that are on the same line as a class are automatically scoped to the class without needing to be enabled again explicitly?  Otherwise is `Style/Documentation` off permanently after Rubocop hits this file?

Reply:
Not quite scoped the the class as mentioned before, but more limits it to problems with the class itself. You were right that this one was redundant though, not sure why Rubocop didn't detect that